src/Form/FormCheckbox.jsx

```diff
@@ -27,6 +27,7 @@ const CheckboxControl = React.forwardRef(
         type="checkbox"
         {...checkboxProps}
         ref={resolvedRef}
+        aria-label="Checkbox"
```

Comment:
[clarification] Should an `aria-label` be added here, without the ability for the consumer to change it? How does this interact with rendering a checkbox with a `FormLabel` (e.g., in `FormCheckbox` below)?

Is this only to resolve when a checkbox is rendered without a label?